app/presenters/hyrax/iiif_manifest_presenter.rb

```diff
@@ -87,7 +87,10 @@ def manifest_url
     ##
     # @return [Array<#to_s>]
     def member_ids
-      Array(model.try(:member_ids))
+      ordered = Array(model.try(:ordered_member_ids))
+      unordered = Array(model.try(:member_ids)) - ordered
+
+      ordered + unordered
```

Comment:
i think this is equivalent, but i like this version for its explicitness. `Array#&`'s order handling won't be very obvious to readers.

```suggestion
      Array(model.try(:ordered_member_ids)) & Array(model.try(:member_ids))
```